indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -1251,12 +1252,19 @@ private TaskStatus runSequential(TaskToolbox toolbox) throws Exception
    */
   private TaskReport.ReportMap getTaskCompletionReports(TaskStatus taskStatus)
   {
-    return buildIngestionStatsAndContextReport(
+    final TaskReport.ReportMap taskCompletionReport = buildIngestionStatsAndContextReport(
         IngestionState.COMPLETED,
         taskStatus.getErrorMsg(),
         segmentsRead,
         segmentsPublished
     );
+    final var totalProcessedBytes = indexGenerateRowStats.lhs.get("processedBytes");
```

Comment:
Since we are going to cast this later in the code anyway, let's just do the cast here and avoid the `var`.
```suggestion
    final Number totalProcessedBytes = (Number) indexGenerateRowStats.lhs.get("processedBytes");
```